lib/resources/df.rb

```diff
@@ -0,0 +1,28 @@
+# encoding: utf-8
+# author: Jonathan Morley
+
+module Inspec::Resources
+  class DfResource < Inspec.resource(1)
+    name 'df'
+    desc 'Use the df InSpec resource to test file system disk space usage'
+    example "
+      describe df('/') do
+        its('size') { should be >= 32000 }
+      end
+    "
+    attr_reader :partition
+
+    def initialize(partition)
+      @partition = partition
+      @cmd = inspec.command("df #{@partition} --output=size | sed \"/blocks/d; s/ *//g\"")
+    end
+
+    def size
+      @cmd.stdout.to_i
+    end
+
+    def to_s
+      "DiskSpace #{@partition}"
```

Comment:
Same here - can just change this to `#{partition}` thanks to it being made into an `attr_reader`